### DISCUSSION TITLE
Error comes when sign in

### DISCUSSION THREAD

**Discussion by Adword104** [community/community#119708] [Mobile]
### Select Topic Area

Bug

### Body

Trying several time from my another account to login on my [pc](https://alightmotionsapps.com/alight-motion-pc/) but it does not work. I tried to make new account but error comes same.
Also try on phone but same error comes.
Please help.
Thank you.

**Comment by stevepopovich** [comment]
Hi @Adword104 do you have more info about this? Could you provide a screenshot of what you see?

**Comment by Mahdi-Shabani** [comment]
Usually, the cause of this error is one of the following:
SSL not enabled
SSL timeout
The system date is not updated
It's problems
